### ISSUE TITLE
debugging of Blazor webassembly does not hit breakpoints

### ISSUE THREAD

**Issue by fabianus76** [dotnet/aspnetcore#22441]
### Describe the bug
 I created a fresh 
dotnet new blazorwasm -o WebApplication1

with .net Core version 3.1.300

then in vs code (with C# extension 1.22.0 and JavaScript Debugger (Nightly) 2020.5.2917)
I mark a breakpoint in the IncrementCount() method. But when hitting the button fire the incrementation, the breakpoint is not hit. 

### To Reproduce

my .vscode/launch.json content:

```
{
   // Use IntelliSense to find out which attributes exist for C# debugging
   // Use hover for the description of the existing attributes
   // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
   "version": "0.2.0",
   "configurations": [
        {
            "name": "Launch and Debug Standalone Blazor WebAssembly App",
            "type": "blazorwasm",
            "request": "launch",
            "cwd": "${workspaceFolder}"
        }
    ]
}
```

my Properties/launchSettings.json content: 

```
{
  "iisSettings": {
    "windowsAuthentication": false,
    "anonymousAuthentication": true,
    "iisExpress": {
      "applicationUrl": "http://localhost:25849",
      "sslPort": 44381
    }
  },
  "profiles": {
    "IIS Express": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
    "WebApplication98": {
      "commandName": "Project",
      "launchBrowser": true,
      "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
      "applicationUrl": "https://localhost:5001;http://localhost:5000",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    }
  }
}
```

and I simply press F5

### Further technical details
- .net Core version 3.1.300
- The IDE VS Code version 1.45.1

**Comment by jongio** [comment]
I noticed the same thing yesterday.

Do you have any other instances of VS Code running?

Are you sure you don't have another VS Code running with the Blazor host running in another terminal?  That was one of the issues for me.

Can you try with JavaScript Debugger: 2020.6.117?  That's the version I'm using and it works (if I have a launch.json and I'm sure all Blazor hosts are killed)

I noticed that when you close the browser, the Host is not stopped:
![image](https://user-images.githubusercontent.com/2163001/83536340-ee233580-a4a7-11ea-8049-48cd0651c27a.png)

I had to go into the terminal and manually stop the host before starting a new debug session.

**Comment by captainsafia** [comment]
@fabianus76 Have you trusted local dev certs before debugging? The [troubleshooting section of the docs](https://docs.microsoft.com/en-us/aspnet/core/blazor/debug?view=aspnetcore-3.1#troubleshoot) has more information on how to do this.

@jongio Are you running into this issue using the new `blazorwasm` debug type? Also, are you on a non-Windows OS?

**Comment by jongio** [comment]
Yes, new debug type.  Windows 10.

**Comment by fabianus76** [comment]
> @fabianus76 Have you trusted local dev certs before debugging? The [troubleshooting section of the docs](https://docs.microsoft.com/en-us/aspnet/core/blazor/debug?view=aspnetcore-3.1#troubleshoot) has more information on how to do this.

Dear @captainsafia - this did it for me. Thanks a lot !

**Comment by captainsafia** [comment]
Glad that resolved your issue -- happy debugging!

@jongio I'll close this issue out for bookkeeping. Can you open a new issue for the problems you were running into and I'll post what we discussed off-GH there?
